src/attr.c

```diff
@@ -218,7 +218,7 @@ int do_attr_command(cmd_request_t cmd)
 	rv = test_attr_reply(ntohl(header->result), cmd);
 
 out_close:
-	if (site)
+	if (tpt && site)
```

Comment:
oops

Reply:
good catch!

Reply:
Btw. there are some chances that with -O2 and higher, this problem is,
very speculatively, masked out, but it will manifest with `--enable-debug`
for sure.

Possible test case:

```
$ arbitrator=127.0.0.1 > /tmp/booth.conf
$ geostore delete -c /tmp/booth.conf -t someticket -s 127.0.0.1 foo
```